src/Symfony/Component/Ldap/LdapClientInterface.php

```diff
@@ -18,12 +18,11 @@
  *
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  * @author Charles Sarrazin <charles@sarraz.in>
+ *
+ * @internal
  */
 interface LdapClientInterface
 {
-    const LDAP_ESCAPE_FILTER = 0x01;
-    const LDAP_ESCAPE_DN = 0x02;
```

Comment:
not sure about this change: using the ext-ldpa constants binds the interface to the extension, thus removes any genericness don't you think?